### PR TITLE
fix(rcha and evta): irch and ievt arrays now zero based

### PR DESCRIPTION
**Pull request by spaulins-usgs**
fix for issue #941.  also changed createpackages.py to always produce python scripts with unix-style line endings.

### CODE REVIEW

**Comment by spaulins-usgs** [comment]
@langevin-usgs not sure why t037_test.test_mf2005swi2load is failing on python 3.8.  The code changed in this PR is exclusively MF6 code.  t037 test is not a MF6 test and should not be executing any of the code changed in this PR.  Maybe try restarting Travis?

File: flopy/mf6/data/dfn/gwf-evta.dfn [hunk 1]
```diff
@@ -157,6 +157,7 @@ name ievt
 type integer
 shape (ncol*nrow; ncpl)
 reader readarray
+numeric_index true
```
Comment (langevin-usgs) [inline flopy/mf6/data/dfn/gwf-evta.dfn:160]: @spaulins-usgs, this is not the authoritative place to make this change.  Have you made this change in the mf6 repo also?

File: flopy/mf6/data/dfn/gwf-rcha.dfn [hunk 2]
```diff
@@ -157,6 +157,7 @@ name irch
 type integer
 shape (ncol*nrow; ncpl)
 reader readarray
+numeric_index true
```
Comment (langevin-usgs) [inline flopy/mf6/data/dfn/gwf-rcha.dfn:160]: same comment here as for gwf-evta.dfn

File: flopy/mf6/data/dfn/gwf-evta.dfn (hunk 1 above)
Comment (spaulins-usgs) [reply flopy/mf6/data/dfn/gwf-evta.dfn:160]: This change is made in my local copy of the mf6 repo.  I will start a mf6 repo PR now.

**Pull request merged**